replication_report-extensions.Rmd

```diff
@@ -0,0 +1,320 @@
+
+```{r eval= FALSE}
+
+# Load data
+dat <- read.csv("GORN_study3.csv", header = TRUE)
+dat <- tbl_df(dat)
+#head(dat)
+#View(dat)
+
+# Designate factors:
+dat$Groep <- as.factor(dat$Groep)
+dat$music <- as.factor(dat$music)
+dat$chose_advertised_pen <- as.factor(dat$chose_advertised_pen)
+
+# Filter out 2 excluded participants who accidentily took study twice in the paper:
+dat_e <- dat %>%
+  filter(excluded == 0)
+#dat_e
+#View(dat_e)
+
+## Analyses on ENTIRE sample:
+
+# Test whether music condition is independent of pen choice:
+# (1 == liked music, 2 == disliked music)
+# (0 == not chosen, 1 == chosen)
+tbl <- table(dat_e$music, dat_e$chose_advertised_pen) 
+tbl    
+
+# Chi-Square, no correction, no simulation:
+cs1 <- chisq.test(tbl, correct = FALSE)
+
+# Power analysis -- Find the power to detect .35 effect size reported in paper:
+pwr.chisq.test(w = .39, N = 91, df = 1, sig.level = 0.05, power = NULL)
+
+# Power analysis -- Find the effect size for a test with .8 sensitivity:
+pwr.chisq.test(w = NULL, N = 91, df = 1, sig.level = 0.05, power = .8)
+
+# Analyses on SUBSETTED sample:
+
+# Filter out additional 19 Ss with too positive attitudes in "disliked music" condition & too negative attitudes in "liked music" condition:
+
+# Find Cronbach's Alpha for music attitudes scale (for comparison with article):
+#dat_music <- dat_e %>%
+#  select(att_muz1, att_muz2, att_muz3)
+#View(dat_music)
+#alpha(dat_music, na.rm = TRUE, delete=TRUE)
+
+# Find mean attitudes for each music condition (for comparison with article): 
+#dat_e %>%
+#  group_by(music) %>%
+#  summarise(avg = mean(c(att_muz1, att_muz2, att_muz3)))
+
+# Add mean attitude column to dataframe:
+dat_e <- dat_e %>%
+  mutate(avg_att_muz = (att_muz1 + att_muz2 + att_muz3) / 3)
+#View(dat_e)
+
+# Check those means (for comparison with article):
+#dat_e %>%
+#  group_by(music) %>%
+#  summarise(avg = mean(avg_att_muz))
+
+# Exclude when average attitude > 3 in disliked cond. or average attitude <3 for liked cond. (1 == liked music, 2 == disliked music)
+dat_egood <- dat_e %>%
+  filter(music == 1 & avg_att_muz > 3 | music == 2 & avg_att_muz < 3)
+#View(dat_egood)
+
+## Redo Chi-Square with subsetted data:
+
+# Test whether music condition is independent of pen choice:
+# Create 2x2 contingency table of frequencies - pen choice by music condition
+tbl2 <- table(dat_egood$music, dat_egood$chose_advertised_pen) 
+tbl2    
+
+# Chi-Square, no correction, no simulation:
+cs2 <- chisq.test(tbl2, correct = FALSE)
+tidy(cs2)
+
+# Power analysis -- Find the power to detect .35 effect size reported in paper:
+pwr.chisq.test(w = .35, N = 72, df = 1, sig.level = 0.05, power = NULL)
+
+# Power analysis -- Find the effect size for a test with minimum of .8 sensitivity:
+pwr.chisq.test(w = NULL, N = 72, df = 1, sig.level = 0.05, power = .8)
+
+# Confidence Intervals (not replicated from paper):
+#upper (95%)
+#CI95_up <- qchisq(.95, df = 1)
+#lower (5%)
+#CI95_low <- qchisq(.05, df = 1)
```

Comment:
I found the best way to replicate 95% confidence intervals was just to multiply 1.96 * the SEM., because if you think about what the 95% confidence interval is, it's 1.96 standard deviations of the sampling distribution (in both directions).  When we tried using a baked in function in R, it was always wrong, but 1.96 * SEM always worked.